app/controllers/api/v1/projects_controller.rb

```diff
@@ -34,6 +34,11 @@ def show
     render json: Api::V1::ProjectSerializer.new(@project, @options)
   end
 
+  def image_preview
+    @project = Project.open.find(params[:id])
```

Comment:
What about private projects? They don't have preview images ?

Reply:
They do have but @satu0king asked me for public projects only, is that right @satu0king ?

Reply:
Yes idea is that we don't want private project's preview image to get leaked this way. @tachyons is there a better way or its fine to do that?

Reply:
@satu0king We can return 403 if the user is not authorized to view this project, right?